<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosConv</class>
 <widget class="QWidget" name="PosConv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POSCONV - Position Log Converter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="paramLayout">
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QRadioButton" name="calculateButton">
         <property name="text">
          <string>Ca&amp;lculate arena frame</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="simulateButton">
         <property name="text">
          <string>Sim&amp;ulate arena frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="pulsesPerRotationLabel">
           <property name="text">
            <string>Pulses per rotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="pulsesPerRotationBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="rpmLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Seconds per revolution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rpmBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileLayout">
     <item>
      <widget class="QListWidget" name="fileList">
       <item>
        <property name="text">
         <string>Add files...</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addDirButton">
         <property name="text">
          <string>Add directory</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear list</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="processButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
